### ISSUE TITLE
0.8.1 release

### ISSUE THREAD

**Issue by adamralph** [adamralph/liteguard#77]
**Ready** when all other issues forming part of the release are **Done**.
- [ ] ~~run code analysis in VS in _Release_ mode and address violations (send a regular PR which must be merged before continuing)~~
- [x] check build, create release in GitHub UI including releaseNotes, mentioning non-owner contributors, if any
- [x] push nuget package
- [x] copy release notes from GitHub to NuGet
- ~~[ ] de-list pre-release nuget packages if present~~
- ~~[ ] update website with contributors list (if in place)~~
- ~~[ ] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)~~
- ~~[ ] post tweet in JabbR (general-chat)~~
- [x] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [ ] ~~increment minor version~~
- [ ] ~~push to origin branch, create PR to upstream master~~
- ~~[ ] check build, merge PR~~
- ~~[ ] create a new milestone for the next release~~
- ~~[ ] create new issue based on this one for next release, adding it to the new milestone~~
- [x] close all issues on this milestone
- [x] close this milestone
